.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
```

Comment:
this is just creating PRs right?
because we have seen the gh actions can have regressions 😉

Reply:
yes, it will create a PR